packages/reakit/src/Form/FormState.ts

```diff
@@ -264,7 +264,7 @@ function reducer<V>(
       };
     }
     case "remove": {
-      const array = unstable_getIn(state.values, action.name, []);
+      const array = unstable_getIn(state.values, action.name, []).slice();
```

Comment:
The reason for the slice confused me for a while as I was expecting getIn to return a copy, like the Immutable.js method. Should this be the case?

Reply:
This is more like lodash's `get`, which doesn't return a copy. In fact, we could even use lodash there and things would still work.

This method is used in other places to get fields values. If it returned a different copy on every render I guess that would cause unnecessary re-renders.

Reply:
Okay, cool!